src/triage/cli.py

```diff
@@ -120,6 +122,10 @@ def __init__(self, parser):
             type=argparse.FileType('r'),
             help="config file for Experiment"
         )
+        parser.add_argument(
+            '-s', '--setup',
+            help="python module to import before running the Experiment",
```

Comment:
To pile on :smile_cat: … this is a bit ambiguous -- is it simply the module's importable name, or its python import path?

(*e.g.* it _could_ be `sfpd.experiment.setup`.)

perhaps:

> file path to Python module to import before running the Experiment